### PR TITLE
Fixed testCreateTable and testAlterTable in CommandTest for MSSQL and OCI

### PR DESCRIPTION
**Pull request by edgardmessias**

### CODE REVIEW

File: tests/framework/db/CommandTest.php [hunk 1]
```diff
@@ -314,7 +314,11 @@ public function testInsertExpression()
     public function testCreateTable()
     {
         $db = $this->getConnection();
-        $db->createCommand("DROP TABLE IF EXISTS testCreateTable;")->execute();
+
+        try {
+            $db->createCommand()->dropTable('testCreateTable')->execute();
+        } catch (\Exception $ex) {
```
Comment (samdark) [inline tests/framework/db/CommandTest.php]: In case of empty `catch` it's good to add comment about the fact that it's on purpose.

**Comment by edgardmessias** [comment]
Added comment

**Comment by cebe** [comment]
if this is MSSQL and OCI specific, it should be overrridden in OracleCommandTest.php and MssqlCommandTest.php.

**Comment by samdark** [comment]
https://github.com/yiisoft/yii2/commit/2d8106a39c586ef61539fd4cbd16964486ed9e02

**Pull request closed without merging**